cluster/addons/podsecuritypolicies/privileged.yaml

```diff
@@ -1,23 +1,24 @@
 apiVersion: extensions/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  annotations:
-    kubernetes.io/description: 'privileged allows access to all privileged and host
-      features and the ability to run as any user, any group, any fsGroup, and with
-      any SELinux context.'
-  creationTimestamp: 2016-05-06T19:28:58Z
   name: privileged
+  annotations:
+    kubernetes.io/description: 'privileged allows full unrestricted access to
+      pod features, as if the PodSecurityPolicy controller was not enabled.'
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
 spec:
   privileged: true
-  defaultAddCapabilities: null
-  requiredDropCapabilities: null
-  allowedCapabilities: null
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - '*'
```

Comment:
This won't work until we merge https://github.com/kubernetes/kubernetes/pull/51337 Probably this PR should depends on it.

Reply:
Thanks for the heads up. I'm not sure why I thought '*' would work here...